lib/ios-sim.ts

```diff
@@ -14,3 +14,39 @@ var fiber = Fiber(() => {
 });
 
 fiber.run();
+
+global.publicApi = {};
+
+Object.defineProperty(global.publicApi, "getRunningSimulator", {
```

Comment:
these are properties and should be named as nouns, not verbs. Consider naming it `runningSimulator`